tests/testthat/test-chunks.R

```diff
@@ -40,3 +40,74 @@ test_that("produce informative errors for bad chunk_vars", {
   }, paste("chunk_vars must be a character vector;",
            "use chunk_vars=character() to specify 1 chunk"), fixed=TRUE)
 })
+
+data(breakpoints)
+only.error <- subset(breakpoints$error,type=="E")
+only.segments <- subset(only.error,bases.per.probe==bases.per.probe[1])
+signal.colors <- c(estimate="#0adb0a",
+                   latent="#0098ef")
+breakpointError <- 
+  list(signal=ggplot()+
+         geom_point(aes(position, signal, showSelected=bases.per.probe),
+                    data=breakpoints$signals)+
+         geom_line(aes(position, signal), colour=signal.colors[["latent"]],
+                   data=breakpoints$imprecision)+
+         geom_segment(aes(first.base, mean, xend=last.base, yend=mean,
+                          showSelected=segments,
+                          showSelected2=bases.per.probe),
+                      colour=signal.colors[["estimate"]],
+                      data=breakpoints$segments)+
+         geom_vline(aes(xintercept=base,
+                        showSelected=segments,
+                        showSelected2=bases.per.probe),
+                    colour=signal.colors[["estimate"]],
+                    linetype="dashed",
+                    data=breakpoints$breaks),
+       error=ggplot()+
+         geom_vline(aes(xintercept=segments, clickSelects=segments),
+                    data=only.segments, lwd=17, alpha=1/2)+
+         geom_line(aes(segments, error, group=bases.per.probe,
+                       clickSelects=bases.per.probe),
+                   data=only.error, lwd=4))
+
+bytes.used <- function(file.vec, ...){
+  ## Note: the --apparent-size flag gives sizes that are consistent
+  ## with file.info, but those sizes actually under-estimate the
+  ## actual amount of space used on disk.
+  file.str <- paste(file.vec, collapse=" ")
+  cmd <- paste("du --block-size=1", ..., file.str)
+  du.lines <- system(cmd, intern=TRUE)
+  as.integer(sub("\t.*", "", du.lines))
+}
+
+test.paths <- 
+  c(tempfile=tempfile(),
+    HOME=file.path(Sys.getenv("HOME"), "ANIMINT_TEST_FOO"),
+    getwd=file.path(getwd(),"ANIMINT_TEST_FOO"))
+for(f in test.paths){
+  unlink(f)
+  cat("foo", file=f)
+}
+du.bytes <- bytes.used(test.paths)
+apparent.bytes <- bytes.used(test.paths, "--apparent-size")
+byte.df <- data.frame(du.bytes, apparent.bytes,
+                      file.size=file.size(test.paths),
+                      test.paths)
+print(byte.df)
```

Comment:
@tdhock could you add a check for `du` (possibly via `Sys.which("du")`) and exit gracefully if it isn't found?